pkg/asset/installconfig/ovirt/credentials.go

```diff
@@ -0,0 +1,66 @@
+package ovirt
+
+import (
+	"gopkg.in/AlecAivazis/survey.v1"
+
+	"github.com/openshift/installer/pkg/types/ovirt"
+)
+
+func askCredentials() (ovirt.Config, error) {
+	c := ovirt.Config{}
+	err := survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Input{
+				Message: "Enter oVirt's api endpoint URL",
+				Help:    "oVirt engine api url, for example https://ovirt-engine-fqdn/ovirt-engine/api",
+			},
+			Validate: survey.ComposeValidators(survey.Required),
+		},
+	}, &c.URL)
+	if err != nil {
+		return c, err
+	}
+
+	var ovirtCertTrusted bool
+	err = survey.AskOne(
+		&survey.Confirm{
+			Message: "Is the installed oVirt certificate trusted?",
+			Default: false,
+			Help:    "",
+		},
+		&ovirtCertTrusted,
+		nil)
+	if err != nil {
+		return c, err
+	}
+	c.Insecure = !ovirtCertTrusted
+
+	err = survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Input{
+				Message: "Enter ovirt-engine username",
+				Help:    "The user must have permissions to create VMs, Disks, on the Storage Domain with the name matching the Openshift cluster",
```

Comment:
The user must have permissions to create VMs and disks on the Storage Domain with the same name as the Openshift cluster",

Reply:
done